# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/nsHttpChannel.h
# Commit: aafdd02eb30a
# Full Hash: aafdd02eb30a2dae68eefb1505083254634d8034
# Author: Paul Bone <pbone@mozilla.com>
# Date: 2019-11-11 21:52:52
# Regressor Bug: 1588412
# File Overlap Count: 2
# Description:
#   Bug 1588412 - Fix a spelling error r=mattwoodrow
#   
#   The already-landed patches contained a spelling error.  This patch fixes it.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D51390
# ==============================================================================

diff -r aab04214324f -r aafdd02eb30a netwerk/protocol/http/nsHttpChannel.h
--- a/netwerk/protocol/http/nsHttpChannel.h	Mon Nov 11 08:25:33 2019 +0000
+++ b/netwerk/protocol/http/nsHttpChannel.h	Mon Nov 04 21:45:10 2019 +0000
@@ -334,7 +334,7 @@
   MOZ_MUST_USE nsresult ContinueProcessResponse2(nsresult);
 
  private:
-  void AssertNotDocumentChennel();
+  void AssertNotDocumentChannel();
 
  public:
   void UpdateCacheDisposition(bool aSuccessfulReval, bool aPartialContentUsed);
